FAQ.md

```diff
@@ -59,6 +70,16 @@ conda activate <my_new_env>
 Installing into a new environment avoids forcing conda to satisfy the dependencies of both the toolkit and all existing packages in that environment.
 Taking the approach that conda environments are generally disposable, even ephemeral, minimizes the chances for hard-to-diagnose dependency issues.
 
+## My conda installation of the toolkit STILL doesn't appear to work. 
+
+Many of our users encounter issues that are ultimately due to their terminal finding a different `conda` at higher priority in their `PATH` than the `conda` deployment where OpenFF is installed. To fix this, find the conda deployment where OpenFF is installed. Then, if that folder is something like `~/miniconda3`, run in the terminal:
+
+```shell
+source ~/miniconda3/etc/profile.d/conda.sh
+```
+
+and then try rerunning and/or reinstalling the Toolkit. 
+
```

Comment:
This makes me think that it might be best to describe how to check if you already have conda installed in the installation guide.

Reply:
Conda is such a deep topic that I'm not actually sure we should to do this - The scope of that explanation could get way too wide ("here's what it means if your conda is in a system folder... it's not a showstopper but it is way more complicated in the following ways"), or have us suggest things that break someone else's computing environment (most of the bashrcs I find in the wild are clearly not understood by the user and have geologic layers of cruft). I like the `conda.sh` script because it's a "steamroll my current terminal session environment and just make it work" command that doesn't depend on the user's system details.